src/libicalvcard/vcardrestriction.c.in

```diff
@@ -90,6 +90,69 @@ int vcardrestriction_compare(vcardrestriction_kind restr, int count)
     return compare_map[restr][count];
 }
 
+/* Special case routines */
+
+static const char *vcardrestriction_validate_datetime_value(
+    const vcardrestriction_record *rec,
+    vcardcomponent *comp, vcardproperty *prop)
+{
+    vcardtimetype t = vcardproperty_get_bday(prop);
+#define TMP_BUF_SIZE 1024
```

Comment:
> nit: didn't we decide to start using static const instead of defines?

GCC doesn't like using a static const size_t as the size of an array:

`/home/ken/libical/src/libicalvcard/vcardrestriction.c:102:17: error: storage size of ‘buf’ isn’t constant
  102 |     static char buf[TMP_BUF_SIZE];
      |                 ^~~
/home/ken/libical/src/libicalvcard/vcardrestriction.c:102:17: error: unused variable ‘buf’ [-Werror=unused-variable]
/home/ken/libical/src/libicalvcard/vcardrestriction.c: In function ‘vcardrestriction_validate_timestamp_value’:
/home/ken/libical/src/libicalvcard/vcardrestriction.c:140:17: error: storage size of ‘buf’ isn’t constant
  140 |     static char buf[TMP_BUF_SIZE];
`